mapping/requirements_officejs.json

```diff
@@ -1036,6 +1036,28 @@
 							}],
 							"availability": "GA"
 						},
+						{
+							"name": "CustomFunctionsRuntime",
+							"apiVersion": "1.2",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.35.127.0",
+									"version": null
+								}
+							}],
+							"availability": "GA"
+						},
+						{
+							"name": "CustomFunctionsRuntime",
+							"apiVersion": "1.3",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.40.810.0",
```

Comment:
Same as above -- Please make Macos, iOS versions default to 0 for last 2 bits
16.35.0.0